andela/ah-leagueOfLegends
# #159965463. Write unit tests for ALL functionality

## What does this PR do?

Implements tests for Authors Haven Functionality.

## Description of tasks to be completed

This PR implements tests for Authors Haven authentication functionality.

## How should this be manually tested?

Clone the repo to the desktop, check out ch-tests-159965463. Run Python manage.py test.
All the tests should pass.

## Relevant PT story

## 159965463
